src/manifest.ts

```diff
@@ -30,7 +30,7 @@ const generateManifest = ({ bundles = [], declarations = [], functions }: Genera
       return
     }
 
-    const pattern = 'pattern' in declaration ? new RegExp(declaration.pattern) : globToRegExp(declaration.path)
```

Comment:
wait what is this `'pattern' in declaration`? is that the same as `declaration?.pattern`? I almost never see `in` used

Reply:
Yes, but since `declaration` is a union type (`DeclarationWithPath` or `DeclarationWithPattern`) and the `pattern` only exists in one of them, the `in` allows us to disambiguate. So if you do:

```ts
if ('pattern' in declaration) {
  // From this point on you know that `declaration` is of type
  // `DeclarationWithPattern` and so does the TypeScript compiler.
}
```